content/blog/ciwms130.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "QGIS Grant Program: QGIS Server, OGC tests and continuous integration"
+date: 2021-02-18
+author: Paul Blottiere
+draft: false
+---
+
+
+# Introduction
+
+Every year, [QGIS.org](https://qgis.org/en/site/) provides funding for
+improving the QGIS ecosystem on too often neglected subjects like refactoring,
+packaging or upstream studies. In light of this event, called QGIS Grant
+Program, some ideas proposed by the
+[QCooperative](https://www.qcooperative.net/) team have been accepted by the
+voting members. One of them was: *QGIS Server, OGC tests and Continuous
```

Comment:
Here we can list the three accepted proposals. Later we can add links to their articles.